README.md

```diff
@@ -204,7 +204,11 @@ Make sure that you have a Vulkan ready driver and install the [LunarG Vulkan SDK
 
 #### Linux
 
-Make sure that you have a Vulkan ready driver and install the [LunarG Vulkan SDK](https://lunarg.com/vulkan-sdk/). You also have to add the library and layers to your path. Have a look at my [post](http://askubuntu.com/a/803110/77183) if you are unsure how to do that.
+Install a Vulkan driver for your graphics hardware of choice, and the [LunarG Vulkan SDK](https://lunarg.com/vulkan-sdk/) for the [ICD](https://github.com/KhronosGroup/Vulkan-Loader) and [Validation Layers](https://github.com/KhronosGroup/Vulkan-ValidationLayers).
```

Comment:
`Installing the LunarG SDK manually is not usually required.` 
`Linux users should typically not "install the SDK", especially not via the provided link; they should install specific distro-provided packages.`
I think I understand the reasoning behind this but,
I don't think all the tools from `LunarG SDK` are available as a package. (I couldn't find vkconfig on Ubuntu)
More over installing the LunarG SDK on linux is can be as simple as download, extract, run `source setup-env.sh`.
Or more to persistently, `echo 'source path_to_sdk/setup-env.sh'`
All it does it setup some environment variables which will work on most distributions that don't have very exotic configurations.

Unless I'm missing something, should that not be the first option since it'll probably work on most distributions?

Reply:
> Unless I'm missing something, should that not be the first option since it'll probably work on most distributions?

It actually won't, using a package manager is always preferable (proper dependency resolution, package tailored to distro-specifics, automatic updates, ...).